### ISSUE TITLE
Card to Card attachments/links

### ISSUE THREAD

**Issue by ricardoboss** [wekan/wekan#3257]
Hi! Thanks for maintaining this project! I'd like to know if this Trello-feature is planned: attaching cars to other cards. When I import a board from Trello, these attached cards get converted to files(?) and are no longer properly linked.

![image](https://user-images.githubusercontent.com/6266356/92242451-157f0980-eec0-11ea-806e-9b6abf33ad33.png)

**Comment by xet7** [comment]
@ricardoboss 

Is it like converting those to some kind of subcards?

Or is it like linked cards? https://github.com/wekan/wekan/pull/1592

I did not even know Trello has this kind of feature.

Wekan has code from about 180 contributors from all over the world. Usually when someone likes to have a feature, it is easiest to just look at source code and add some new code, then send pull request. For example, some other Wekan contributor recently added voting, and importing votes from Trello. Mostly it's up to Wekan Community at these GitHub issues, about what features Wekan should have, what they send pull requests for, etc. I don't know everyones use cases.

For example in this case, look at Trello exported JSON, and code at this repos https://github.com/wekan/wekan at directory wekan/models/*import* and add some code to import that to Wekan database.

I am only aware of those prepaid features at my Roadmap, link to Roadmap is at Wekan website https://wekan.github.io , those have been prepaid at my [Commercial Support page](https://wekan.team/commercial-support/) by those that really would like that my maintaining and adding features and fixes continues.

**Comment by xet7** [comment]
In general, most Trello features will be added to Wekan, so that importing from Trello would work better.

**Comment by MacOS** [comment]
Hi all,

is this feature still open, in the sense that you are waitingn for someone to implement it?

**Comment by xet7** [comment]
@MacOS 

Yes. Anyone can [send PR](https://github.com/wekan/wekan/wiki/Emoji) for any feature or fix.

I'm also adding features and fixes to WeKan, but if other WeKan contributors help, we get features and fixes working much earlier.

**Comment by MacOS** [comment]
Great! Then I would be interested in implementing it.

**Comment by MacOS** [comment]
Thank you, @xet7! Can you please assign me to the issue so it shows up on my GitHub page? Thank you!
